How to Contribute to ETC: Mining (#1164)
ethereumclassic/ethereumclassic.github.io: content/blog/2023-04-19-how-to-contribute-to-etc-mining/index.md

As nearly all the other smart contract blockchains in the market are proof of stake, a much less secure consensus mechanism that tends to centralization, ETC may be considered the most secure programmable decentralized platform in the world.

The above, coupled with its strong core principles and its capped supply, promise an incredible potential for the ETC. 

This potential may be harvested by miners as the business itself is a way of acquiring ETC at discounted prices due to its profitability.

## Ethereum Classic’s Mining Algorithm

One of Ethereum Classic’s unique features is the ETC Hash mining algorithm.

As Ethereum was using the ETH Hash algorithm before it migrated to proof of stake, Ethereum Classic had to differentiate its mining format to defend itself from 51% attacks that came from the Ethereum computing base. This led to the creation of ETC Hash.

> ETC Hash is a modification of ETH Hash where one component, the DAG, which is a large file that must be used during the mining process, was reduced in size and its size growth was slowed to permit smaller GPU miners; with GPU card capacities that had lower memories, such as 3GB, 4GB, and 6GB; mine and stay mining in Ethereum Classic for a longer period of time.

Reviewer: Line 70: Rewrite for structure and clarity
```suggestion
ETC Hash is a modification of ETH Hash wherein one component, the DAG, a large file that must be used during the mining process, was reduced in size and its growth slowed to permit GPU miners with 3GB, 4GB, and 6GB; of memory per GPU card to mine Ethereum Classic for a longer period of time before DAG growth exceeds their hardware limitations..
```

The above is a significant benefit for smaller miners who may have these machines or may acquire them cheaply in the secondary market.

## Mining Gear

Ethereum Classic may be efficiently mined with ASIC machines, which are especially configured computers to exclusively mine the ETC Hash and ETH Hash algorithms, or GPU machines which are regular graphics processing computing devices that may mine ETC with the correct software and configuration. ETC is very inefficient to mine with CPU machines.

Because the market has become very competitive, it is advisable to acquire ASIC machines to mine ETC. We have reviewed the [iPollo V1 Mini SE Plus ASIC miner](https://ethereumclassic.org/blog/2023-03-14-mining-ethereum-classic-with-an-ipollo-asic-through-2miners-pool) and will soon publish a post about the Bitmain Antminer E9 ASIC miner.

> These mining machines may be acquired in the following links:

Reviewer: Line 80: Usage correction
```suggestion
These mining machines may be acquired from the following links:
```

These mining machines may be acquired from the following links:

**Buy iPollo:** https://ipollo.com/products/ipollo-v1-mini-se-plus

**Buy Antminer:** https://bt-miners.com/products/bitmain-antminer-e9

## Mining Pools

Mining pools are companies that aggregate many miners around the world who provide their computing power to produce blocks. 

The benefit of mining pools is that they smooth out the cash flows for miners because blocks are found more frequently by participating in a larger computing base and they distribute the payments evenly in time. 

In exchange for this service mining pools charge a percentage of the reward payments as a commission that usually ranges between 1% and 3%.

Popular mining pools in ETC are:

**F2pool:** https://f2pool.io/mining/guides/how-to-mine-ethereum-classic/

**2miners:** https://etc.2miners.com/

**Ethermine:** https://etc.ethermine.org/

**Poolin:** https://help.poolin.me/hc/en-us/articles/5027017873689-Poolin-Supports-ETC-Mining

**Antpool:** https://support.antpool.com/hc/en-us/articles/5984283054745-ETC-Mining-Instruction

> *Disclaimer: The fact that we mention some mining gear and mining pool brands and services in this article does not necessarily constitute or imply their endorsement, recommendation, or favoring by this website or its administrators. We have no commercial relationship with these brands and we do not perceive any fees or affiliate marketing commissions. Do your own reserach before acquiring mining gear or using a mining pool.*

Reviewer: Line 106: Correct usage; Correct spelling
```suggestion
*Disclaimer: The fact that we mention some mining gear and mining pool brands and services in this article does not necessarily constitute or imply their endorsement, recommendation, or favoring by this website or its administrators. We have no commercial relationship with these brands and we do not receive any fees or affiliate marketing commissions. Do your own research before acquiring mining gear or using a mining pool.*
```
